src/addition.py

```
@@ -1,5 +1,7 @@
 # app.py
 # This is a test commit
+#Please get all thing
+update
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)